README.md

```diff
@@ -1,2 +1,102 @@
-# steampipe-plugin-cloudformation
-Use SQL to instantly query resources, data sources and more from CloudFormation templates. Open source CLI. No DB required.
+![image](https://hub.steampipe.io/images/plugins/turbot/awscfn-social-graphic.png)
+
+# AWS CloudFormation Plugin for Steampipe
+
+Use SQL to query data from AWS CloudFormation template files.
+
+- **[Get started →](https://hub.steampipe.io/plugins/turbot/awscfn)**
+- Documentation: [Table definitions & examples](https://hub.steampipe.io/plugins/turbot/awscfn/tables)
+- Community: [Slack Channel](https://steampipe.io/community/join)
+- Get involved: [Issues](https://github.com/turbot/steampipe-plugin-awscfn/issues)
+
+## Quick start
+
+Install the plugin with [Steampipe](https://steampipe.io):
+
+```shell
+steampipe plugin install awscfn
+```
+
+Configure your [config file](https://hub.steampipe.io/plugins/turbot/awscfn#configuration) to include directories with AWS CloudFormation template files. If no directory is specified, the current working directory will be used.
+
+Run steampipe:
+
+```shell
+steampipe query
+```
+
+Query all resources in your AWS CloudFormation template files:
+
+```sql
+select
+  name,
+  type,
+  jsonb_pretty(properties) as resource_properties
+from
+  awscfn_resource;
+```
+
+```sh
+> select name, type, jsonb_pretty(arguments) as args from awscfn_resource;
++-----------+-----------------+---------------------------------------+---------------------------------------+
+| name      | type            | jsonb_pretty                          | jsonb_pretty                          |
```

Comment:
This output doesn't match the query above (and the query doesn't match the query on lines 31-36 either), can you please update this section so everything is consistent?